src/main/java/org/zalando/nakadi/repository/db/SchemaRepository.java

```diff
@@ -28,6 +31,24 @@ public SchemaRepository(final JdbcTemplate jdbcTemplate, final ObjectMapper obje
                 new SchemaRowMapper());
     }
 
+    public EventTypeSchema getSchemaVersion(final String name, final String version)
+            throws NoSuchSchemaException, InternalNakadiException {
+        final String sql = "SELECT ets_schema_object FROM zn_data.event_type_schema " +
+                "WHERE ets_event_type_name = ? AND ets_schema_object ->> 'version' = ?";
+
+        try {
+            final List<EventTypeSchema> schemas =
+                    jdbcTemplate.query(sql, new Object[]{name, version}, new SchemaRowMapper());
```

Comment:
Fixed in https://github.com/zalando/nakadi/commit/b28dac092bd04dd7c9b43f66d7664351359f4ede